# HG changeset patch
# Parent 59369fd64f93346c2622c3063e52d68bb7d38fee
# User Frédéric Wang <fred.wang@free.fr>
Remove old MathML tests that have been moved to MDN. b=585142, r=karlt.

diff --git a/layout/mathml/tests/all-presentation.xml b/layout/mathml/tests/all-presentation.xml
deleted file mode 100644
--- a/layout/mathml/tests/all-presentation.xml
+++ /dev/null
@@ -1,364 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE html PUBLIC
-  "-//W3C//DTD XHTML 1.1 plus MathML 2.0//EN"
-  "http://www.w3.org/TR/MathML2/dtd/xhtml-math11-f.dtd"
-[
-  <!ENTITY mathml "http://www.w3.org/1998/Math/MathML">
-]>
-<html xmlns="http://www.w3.org/1999/xhtml">
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
-<head>
-  <title>Examples of Presentation Markup</title>
-</head>
-<body>
-
-<div>
-Testing tensor indices &lt;mmultiscripts&gt;:
-
-<!-- {3 \atop k} R {1 \atop i} {2 \atop j} -->
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<mmultiscripts>
-  <mi>R</mi>
-  <mi>i</mi><mi>1</mi>
-  <mi>j</mi><mi>2</mi>
-    <mprescripts/>
-  <mi>k</mi><mi>3</mi>
-</mmultiscripts>
-</math>;
-
-This with &lt;none/&gt;, 
-
-<!-- {{} \atop i} A {p \atop q} -->
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<mmultiscripts>
-  <mi>A</mi>
-  <mi>q</mi><mi>p</mi>
-    <mprescripts/>
-  <mi>i</mi><none/>
-</mmultiscripts>
-</math>;
-
-A bit of calculus:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- \int_a^b f(x)dx -->
-<msubsup>
-  <mo>&Integral;</mo>
-  <mi>a</mi>
-  <mi>b</mi>
-</msubsup>
-<mrow>
-<mi>f</mi>
-  <mo>(</mo>
-  <mi>x</mi>
-  <mo>)</mo>
-  <mi>dx</mi>
-</mrow>
-
-<!-- \frac{\partial}{\partial x}F(x,y) + \frac{\partial}{\partial y}F(x,y) -->
-<mrow>
-  <mfrac>
-    <mo>&PartialD;</mo>
-    <mrow>
-      <mo>&PartialD;</mo>
-      <mi>x</mi>
-    </mrow>
-  </mfrac>
-  <mrow>
-    <mi>F</mi>
-    <mo>(</mo>
-    <mi>x</mi>
-    <mo>,</mo>
-    <mi>y</mi>
-    <mo>)</mo>
-  </mrow>  
-  <mo>+</mo>
-  <mfrac>
-    <mo>&PartialD;</mo>
-    <mrow>
-      <mo>&PartialD;</mo>
-      <mi>y</mi>
-    </mrow>
-  </mfrac>
-  <mrow>
-    <mi>F</mi>
-    <mo>(</mo>
-    <mi>x</mi>
-    <mo>,</mo>
-    <mi>y</mi>
-    <mo>)</mo>
-  </mrow>
-</mrow>
-</math>;
-
-Here is the alphabet with invisible
-portions wrapped by &lt;mphantom&gt; in between:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-  <mi>a</mi>
-  <mi>b</mi>
-  <mi>c</mi> 
-<mphantom>
-  <mi>d</mi>
-  <mi>e</mi>
-  <mi>f</mi>
-  <mi>g</mi>
-  <mi>h</mi>
-  <mi>i</mi>
-  <mi>j</mi>
-</mphantom>
-  <mi>k</mi>
-  <mi>l</mi>
-  <mi>m</mi>
-  <mi>n</mi>
-  <mi>o</mi>
-<mphantom>
-  <mi>p</mi>
-  <mi>q</mi>
-  <mi>r</mi>
-  <mi>s</mi>
-</mphantom> 
-  <mi>t</mi>
-  <mi>u</mi>
-  <mi>v</mi>
-  <mi>w</mi>
-  <mi>x</mi>
-  <mi>y</mi>
-  <mi>z</mi>
-</math>.
-
-Testing MathML &lt;msub&gt;:
-
-<!-- a_b -->
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<msub>
-  <mi>a</mi>
-  <mi>b</mi>
-</msub>
-</math>;
-
-<!-- a_i -->
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<msub>
-  <mi>a</mi>
-  <mi>i</mi>
-</msub>
-</math>;
-
-<!-- A_{I_{k}} -->
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<msub>
-  <mi>A</mi>
-  <msub>
-    <mi>I</mi>
-    <mi>k</mi>
-  </msub>
-</msub>
-</math>;
-
-
-Testing MathML &lt;msup&gt;:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- d^b -->
-<msup>
-  <mi>d</mi>
-  <mi>b</mi>
-</msup>
-
-<!-- 2^{a_x} -->
-<msup>
-  <mn>2</mn>
-  <msub>
-    <mi>a</mi>
-    <mi>x</mi>
-  </msub>
-</msup>
-
-<!-- 2^{2^x} -->
-<msup>
-  <msup>
-    <mn>2</mn>
-    <mn>2</mn>
-  </msup>
-  <mi>x</mi>
-</msup>
-
-<!-- {\frac{1}{2}}^{y^{a_x}} -->
-<msup>
-  <mrow>
-    <mo>(</mo>
-    <mfrac>
-      <mn>1</mn>
-      <mn>2</mn>
-    </mfrac>
-    <mo>)</mo>
-  </mrow>
-  <msup>
-    <mi>y</mi>
-    <msub>
-      <mi>a</mi>
-      <mi>x</mi>
-    </msub>
-  </msup>
-</msup>
-</math>.
-
-Testing MathML &lt;munder&gt;, &lt;mover&gt;, and &lt;munderover&gt;:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<munder>
-  <mi>abcd</mi>
-  <mi>un</mi>
-</munder>
-
-<mover>
-  <mi>abcd</mi>
-  <mi>ov</mi>
-</mover>
-
-<munderover>
-  <mi>abcd</mi>
-  <mi>under</mi>
-  <mi>over</mi>
-</munderover>
-</math>.
-
-
-Testing MathML &lt;msubsup&gt;:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- a_b^c -->
-<msubsup>
-  <mi>a</mi>
-  <mi>p</mi>
-  <mi>q</mi>
-</msubsup>
-
-<!-- a_{b+c}^x -->
-<msubsup>
-  <mi>a</mi>
-  <mrow>
-    <mi>b</mi>
-    <mo>+</mo>
-    <mi>c</mi>
-  </mrow>
-  <mi>x</mi>
-</msubsup>
-</math>.
-
-
-Testing MathML &lt;mrow&gt;:
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- d^{a/b} -->
-<msup>
-  <mi>d</mi>
-  <mrow>
-    <mo>(</mo>
-    <mfrac>
-      <mi>a</mi>
-      <mi>b</mi>
-    </mfrac>
-    <mo>)</mo>
-  </mrow>
-</msup>
-</math>;
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- x^2  +  4*x  +  \frac{p}{q} = 0 --> 
-<mrow>
-  <mrow>
-    <msup>
-      <mi>x</mi>
-      <mn>2</mn>
-    </msup>
-    <mo>+</mo>
-    <mrow>
-      <mn>4</mn>
-      <mo>*</mo>
-      <mi>x</mi>
-    </mrow>
-    <mo>+</mo>
-    <mfrac>
-      <mi>p</mi>
-      <mi>q</mi>
-    </mfrac>
-  </mrow>
-  <mo>=</mo>
-  <mn>0</mn>
-</mrow>
-</math>,
-    
-with this &lt;mfrac&gt; hanging here
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<!-- \frac{d*T^{(\frac{i+j}{n})} + p_y*q} 
-          {p^x*b_x + \frac{a+c}{d}} -->
-<mfrac>
-  <mrow>
-    <mi>d</mi>
-    <mo>*</mo>
-    <msup>
-      <mi>T</mi>
-      <mrow>
-        <mo>(</mo>
-        <mfrac>
-          <mrow>
-            <mi>i</mi>
-            <mo>+</mo>
-            <mi>j</mi>
-          </mrow>
-          <mi>n</mi>
-        </mfrac>
-        <mo>)</mo>
-      </mrow>
-    </msup>
-    <mo>+</mo>
-    <msub>
-      <mi>p</mi>
-      <mi>y</mi>
-    </msub>
-    <mo>*</mo>
-    <mi>q</mi>
-  </mrow>
-  <mrow>
-    <msup>
-      <mi>p</mi>
-      <mi>x</mi>
-    </msup>
-    <mo>*</mo>
-    <msub>
-      <mi>b</mi>
-      <mi>x</mi>
-    </msub>
-    <mo>+</mo>
-    <mfrac>
-      <mrow>
-        <mi>a</mi>
-        <mo>+</mo>
-        <mi>c</mi>
-      </mrow>
-      <mi>d</mi>
-    </mfrac>
-  </mrow>
-</mfrac>
-</math>
-
-in the middle of a lot of running text to try to explain what this
-means to those who care to read.
-
-Testing MathML merror, mtext,
-
-<math xmlns="http://www.w3.org/1998/Math/MathML">
-<mtext>This is a text in mtext</mtext>
-<merror><mtext>This is a text in merror</mtext></merror>
-</math>
-
-</div>
-
-</body>
-</html>
diff --git a/layout/mathml/tests/various.xml b/layout/mathml/tests/various.xml
deleted file mode 100644
--- a/layout/mathml/tests/various.xml
+++ /dev/null
@@ -1,679 +0,0 @@
-<?xml version="1.0"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<!DOCTYPE html PUBLIC
-  "-//W3C//DTD XHTML 1.1 plus MathML 2.0//EN"
-  "http://www.w3.org/TR/MathML2/dtd/xhtml-math11-f.dtd"
-[
-  <!ENTITY mathml "http://www.w3.org/1998/Math/MathML">
-]>
-<html xmlns="http://www.w3.org/1999/xhtml">
-
-<head>
-<title>Various examples of MathML</title>
-<style type="text/css">
-maction {
-  background-color: yellow;
-}
-maction:hover  {
-  outline: 1px dotted black;
-/*  border: 1px solid black; */
-}
-</style>
-</head>
-<body>
-
-<h2>&lt;maction&gt;</h2>
-
-<p>Click to toggle between expressions, and watch the status line onmouseover/onmouseout:
-<br />
-
-<math mode="display" xmlns="&mathml;">
-<maction actiontype="toggle">
-  <maction actiontype="statusline">
-    <mi>statusline#First Expression</mi>
-    <mtext>First Expression</mtext>
-  </maction>
-
-  <maction actiontype="statusline">
-    <mi>statusline#Second Expression</mi>
-    <mtext>Second Expression</mtext>
-  </maction>
-
-
-  <maction actiontype="statusline">
-    <mi>statusline#And so on...</mi>
-    <mtext>And so on..</mtext>
-  </maction>
-</maction>
-</math></p>
-
-<p>Click the expression below to see several definitions of pi:
-<br />
-
-<math mode="display" xmlns="&mathml;">
-<mrow>
-<maction actiontype="toggle">
-
-  <mrow>
-    <mi>&pi;</mi> 
-    <mo>=</mo>
-    <mn>3.14159265358</mn><mo mathvariant="bold">...</mo>
-  </mrow>
-
-  <mrow>
-    <mi>&pi;</mi> 
-    <mo>=</mo>
-    <mn>2</mn><mi>i</mi>
-    <mo>&InvisibleTimes;</mo>
-    <mo>Log</mo>
-    <mfrac>
-      <mrow><mn>1</mn><mo>-</mo><mi>i</mi></mrow>
-      <mrow><mn>1</mn><mo>+</mo><mi>i</mi></mrow>
-    </mfrac>
-  </mrow>
-
-  <mrow>
-    <mi>&pi;</mi> 
-    <mo>=</mo>
-    <mn>2</mn>
-    <mphantom><mo>.</mo></mphantom>
-    <mfrac>
-      <mn>2</mn>
-      <msqrt>
-        <mn>2</mn>
-      </msqrt>
-    </mfrac>
-    <mphantom><mo>.</mo></mphantom>
-    <mfrac>
-      <mn>2</mn>
-      <msqrt>
-        <mn>2</mn> 
-        <mo>+</mo> 
-        <msqrt>
-          <mn>2</mn>
-        </msqrt>
-      </msqrt>
-    </mfrac>
-    <mphantom><mo>.</mo></mphantom>
-    <mfrac>
-      <mn>2</mn>
-      <msqrt>
-        <mn>2</mn> 
-        <mo>+</mo> 
-        <msqrt>
-          <mn>2</mn> 
-          <mo>+</mo> 
-          <msqrt>
-            <mn>2</mn>
-          </msqrt>
-        </msqrt>
-      </msqrt>
-    </mfrac>
-    <mo mathvariant="bold">...</mo>
-  </mrow>
-
-  <mrow>
-<mfrac>
-    <mi>&pi;</mi>
-    <mn>4</mn>
-</mfrac>
-    <mo>=</mo>
-  <mfrac>
-     <mstyle scriptlevel="0">
-       <mn>1</mn>
-     </mstyle>
-     <mstyle scriptlevel="0">
-        <mrow>
-          <mn>2</mn>
-          <mo>+</mo>
-          <mfrac>
-            <mstyle scriptlevel="0">
-              <msup><mn>1</mn><mn>2</mn></msup>
-            </mstyle>
-            <mstyle scriptlevel="0">
-               <mrow>
-                 <mn>2</mn>
-                 <mo>+</mo>
-                 <mfrac>
-                   <mstyle scriptlevel="0">
-                     <msup><mn>3</mn><mn>2</mn></msup>
-                   </mstyle>               
-                   <mstyle scriptlevel="0">
-                     <mrow>
-                        <mn>2</mn>
-                        <mo>+</mo>
-                        <mfrac>
-                          <mstyle scriptlevel="0">
-                            <msup><mn>5</mn><mn>2</mn></msup>
-                          </mstyle>
-                          <mstyle scriptlevel="0">
-                            <mrow>
-                               <mn>2</mn>
-                               <mo>+</mo>
-                               <mfrac>
-                                 <mstyle scriptlevel="0">
-                                   <msup><mn>7</mn><mn>2</mn></msup>
-                                 </mstyle>
-                                 <mstyle scriptlevel="0">
-                                   <mn>2</mn><mo>+</mo><mo mathvariant="bold">...</mo>
-                                 </mstyle>
-                               </mfrac>
-                             </mrow>
-                          </mstyle>
-                        </mfrac>
-                      </mrow>
-                   </mstyle>
-                 </mfrac>
-               </mrow>
-            </mstyle>
-          </mfrac>      
-        </mrow> 
-     </mstyle> 
-  </mfrac>  
-  </mrow>
-</maction>
-</mrow>
-</math></p>
-
-<h2>Thomson scattering theory</h2>
-
-<p><math xmlns="&mathml;" mode="display">
-<mrow>
-  <mtable align='left'>
-
-    <mtr>
-      <mtd columnalign='left'>
-        <mrow>
-          <mfrac>
-            <mrow>
-              <msup>
-                <mi>d</mi>
-                <mn>2</mn>
-              </msup>
-              <mi>P</mi>
-            </mrow>
-
-            <mrow>
-              <mi>d</mi>
-              <msub>
-                <mi>&Omega;</mi>
-                <mi>s</mi>
-              </msub>
-              <mo>&ThinSpace;</mo>
-              <mi>d</mi>
-              <msub>
-                <mi>&omega;</mi>
-                <mi>s</mi>
-              </msub>
-            </mrow>
-          </mfrac>
-        </mrow>
-      </mtd>
-      
-      <mtd columnalign='left'>
-        <mrow>
-          <mo>=</mo>
-        </mrow>
-      </mtd>
-      
-      <mtd columnalign='left'>
-        <mrow>
-          <msubsup>
-            <mi>r</mi>
-            <mi>e</mi>
-            <mn>2</mn>
-          </msubsup>
-
-          <msub>
-            <mo>&int;</mo>
-            <mi>V</mi>
-          </msub>
-
-          <mo lspace='0'>&lt;</mo>
-          <msub>
-            <mi>S</mi>
-            <mi>i</mi>
-          </msub>
-          <mo>&gt;</mo>
-
-          <msup>
-            <mi>d</mi>
-            <mn>3</mn>
-          </msup>
-
-          <mi mathvariant='bold'>r</mi>
-          <mo>&int;</mo>
-
-          <msup>
-            <mrow>
-              <mo lspace='0' rspace='0' symmetric='false'>|</mo>
-
-              <mover accent='true'>
-                <mi mathvariant='bold'>e</mi>
-                <mo>&Hat;</mo>
-              </mover>
-
-              <mo>.</mo>
- 
-              <mover accent='true'>
-                <mo>&Pi;</mo>
-                <mo>&#x2194;</mo>
-              </mover>
- 
-              <mo>.</mo>
-
-              <mover accent='true'>
-                <mi mathvariant='bold'>e</mi>
-                <mo>&Hat;</mo>
-              </mover>
- 
-              <mo lspace='0' rspace='0' symmetric='false'>|</mo>
-            </mrow>
-            <mn>2</mn>
-          </msup>
-          
-          <msup>
-            <mi>&kappa;</mi>
-            <mn>2</mn>
-          </msup>
-
-          <mi>f</mi>
-          <mo>&ThinSpace;</mo>
-          <mi>&delta;</mi>
-
-          <mrow>
-            <mo stretchy='false'>(</mo>
-            <mi mathvariant='bold'>k</mi>
-            <mo>.</mo>
-            <mi mathvariant='bold'>v</mi>
-            <mo>-</mo>
-            <mi>&omega;</mi>
-            <mo stretchy='false'>)</mo>
-          </mrow>
-          
-          <msup>
-            <mi>d</mi>
-            <mn>3</mn>
-          </msup>
-          
-          <mi mathvariant='bold'>v</mi>
-        </mrow>
-      </mtd>
-    </mtr>
-    
-    <mtr>
-      <mtd></mtd>
-      
-      <mtd columnalign='left'>
-        <mrow>
-          <mo>=</mo>
-        </mrow>
-      </mtd>
-      
-      <mtd columnalign='left'>
-        <mrow>
-          <msubsup>
-            <mi>r</mi>
-            <mi>e</mi>
-            <mn>2</mn>
-          </msubsup>
-          
-          <msub>
-            <mo>&int;</mo>
-            <mi>V</mi>
-          </msub>
-          
-          <mo lspace='0'>&lt;</mo>
-          <msub>
-            <mi>S</mi>
-            <mi>i</mi>
-          </msub>
-          <mo>&gt;</mo>
-          
-          <msup>
-            <mi>d</mi>
-            <mn>3</mn>
-          </msup>
-
-          <mi mathvariant='bold'>r</mi>
-          <mo>&int;</mo>
-          
-          <msup>
-            <mrow>
-              <mo symmetric='false' lspace='0' rspace='0'>|</mo>
-              <mn>1</mn>
-              <mo>-</mo>
-
-              <mfrac>
-                <mrow>
-                  <mo stretchy='false'>(</mo>
-                  <mn>1</mn>
-                  <mo>-</mo>
-                  <mover accent='true'>
-                    <mi mathvariant='bold'>s</mi>
-                    <mo>&Hat;</mo>
-                  </mover>
-                  <mo>.</mo>
-                  <mover accent='true'>
-                    <mi mathvariant='bold'>&imath;</mi>
-                    <mo>&Hat;</mo>
-                  </mover>
-                  <mo stretchy='false'>)</mo>
-                </mrow>
-
-                <mrow>
-                  <mo stretchy='false'>(</mo>
-                  <mn>1</mn>
-                  <mo>-</mo>
-                  <msub>
-                    <mi>&beta;</mi>
-                    <mi>i</mi>
-                  </msub>
-                  <mo stretchy='false'>)</mo>
-                  <mo stretchy='false'>(</mo>
-                  <mn>1</mn>
-                  <mo>-</mo>
-                  <msub>
-                    <mi>&beta;</mi>
-                    <mi>s</mi>
-                  </msub>
-                  <mo stretchy='false'>)</mo>
-                </mrow>
-              </mfrac>
-              
-              <msubsup>
-                <mi>&beta;</mi>
-                <mi>e</mi>
-                <mn>2</mn>
-              </msubsup>
-
-              <mo symmetric='false' lspace='0' rspace='0'>|</mo>
-            </mrow>
-            <mn>2</mn>
-          </msup>
-
-          <mspace width="thinmathspace"/>
-          
-          <msup>
-            <mrow>
-              <mo symmetric='false' rspace='0'>|</mo>
-              <mfrac>
-
-                <mrow>
-                  <mn>1</mn>
-                  <mo>-</mo>
-                  <msub>
-                    <mi>&beta;</mi>
-                    <mi>i</mi>
-                  </msub>
-                </mrow>
-                
-                <mrow>
-                  <mn>1</mn>
-                  <mo>-</mo>
-                  <msub>
-                    <mi>&beta;</mi>
-                    <mi>s</mi>
-                  </msub>
-                </mrow>
-              </mfrac>
-              
-              <mo symmetric='false' lspace='0' rspace='0'>|</mo>
-            </mrow>
-            <mn>2</mn>
-          </msup>
-        </mrow>
-      </mtd>
-
-    </mtr>
-    
-    <mtr>
-      <mtd></mtd>
-      <mtd></mtd>
-      <mtd columnalign='left'>
-        <mrow>
-          <mo>&times;</mo>
- 
-          <mrow>
-            <mo stretchy='false'>(</mo>
-            <mn>1</mn>
-            <mo>-</mo>          
-            <msup>
-              <mi>&beta;</mi>
-              <mn>2</mn>
-            </msup>       
-            <mo stretchy='false'>)</mo>
-          </mrow>
-          
-          <mo>&ThinSpace;</mo>
-          <mi>f</mi>
-          <mo>&ThinSpace;</mo>
-          <mi>&delta;</mi>
-          
-          <mrow>
-            <mo stretchy='false'>(</mo>
-            <mi mathvariant='bold'>k</mi>
-            <mo>.</mo>
-            <mi mathvariant='bold'>v</mi>
-            <mo>-</mo>
-            <mi>&omega;</mi>
-            <mo stretchy='false'>)</mo>
-          </mrow>
-
-          <msup>
-            <mi>d</mi>
-            <mn>3</mn>
-          </msup>
-          
-          <mi mathvariant='bold'>v</mi>
-        </mrow>
-      </mtd>
-
-    </mtr>
-  </mtable>
-</mrow>
-</math>
-
-</p>
-
-<h2>Maxwell's Equations</h2>
-
-<p style="text-align:center">
-<math xmlns='&mathml;'>
-  <mstyle displaystyle='true'>
-    <mrow>
-      <mo>&lbrace;</mo>
-      <mtable columnalign='right center left' equalrows='false' equalcolumns='false'>
-        <mtr>
-          <mtd>
-            <mrow>
-              <mo>&nabla;</mo>
-              <mo>&times;</mo>
-              <mover accent='true'>
-                <mi mathvariant='bold'>B</mi>
-                <mo stretchy='true'>&RightArrow;</mo>
-              </mover>
-              <mo>-</mo>
-              <mfrac>
-                <mn>1</mn>
-                <mi>c</mi>
-              </mfrac>
-              <mfrac>
-                <mrow>
-                  <mo>&PartialD;</mo>
-                  <mover accent='true'>
-                    <mi mathvariant='bold'>E</mi>
-                    <mo stretchy='true'>&RightArrow;</mo>
-                  </mover>
-                </mrow>
-                <mrow>
-                  <mo>&PartialD;</mo>
-                  <mi>t</mi>
-                </mrow>
-              </mfrac>
-            </mrow>
-          </mtd>
-          <mtd>
-            <mo>=</mo>
-          </mtd>
-          <mtd>
-            <mrow>
-              <mfrac>
-                <mrow>
-                  <mn>4</mn>
-                  <mi>&pi;</mi>
-                </mrow>
-                <mi>c</mi>
-              </mfrac>
-              <mover accent='true'>
-                <mi mathvariant='bold'>j</mi>
-                <mo stretchy='true'>&RightArrow;</mo>
-              </mover>
-            </mrow>
-          </mtd>
-        </mtr>
-        <mtr>
-          <mtd>
-            <mrow>
-              <mo>&nabla;</mo>
-              <mo>&cdot;</mo>
-              <mover accent='true'>
-                <mi mathvariant='bold'>E</mi>
-                <mo stretchy='true'>&RightArrow;</mo>
-              </mover>
-            </mrow>
-          </mtd>
-          <mtd>
-            <mo>=</mo>
-          </mtd>
-          <mtd>
-            <mrow>
-              <mn>4</mn>
-              <mi>&pi;</mi>
-              <mi>&rho;</mi>
-            </mrow>
-          </mtd>
-        </mtr>
-        <mtr>
-          <mtd>
-            <mrow>
-              <mo>&nabla;</mo>
-              <mo>&times;</mo>
-              <mover accent='true'>
-                <mi mathvariant='bold'>E</mi>
-                <mo stretchy='true'>&RightArrow;</mo>
-              </mover>
-              <mo>+</mo>
-              <mfrac>
-                <mn>1</mn>
-                <mi>c</mi>
-              </mfrac>
-              <mfrac>
-                <mrow>
-                  <mo>&PartialD;</mo>
-                  <mover accent='true'>
-                    <mi mathvariant='bold'>B</mi>
-                    <mo stretchy='true'>&RightArrow;</mo>
-                  </mover>
-                </mrow>
-                <mrow>
-                  <mo>&PartialD;</mo>
-                  <mi>t</mi>
-                </mrow>
-              </mfrac>
-            </mrow>
-          </mtd>
-          <mtd>
-            <mo>=</mo>
-          </mtd>
-          <mtd>
-            <mover accent='true'>
-              <mn mathvariant='bold'>0</mn>
-              <mo stretchy='true'>&RightArrow;</mo>
-            </mover>
-          </mtd>
-        </mtr>
-        <mtr>
-          <mtd>
-            <mrow>
-              <mo>&nabla;</mo>
-              <mo>&cdot;</mo>
-              <mover accent='true'>
-                <mi mathvariant='bold'>B</mi>
-                <mo stretchy='true'>&RightArrow;</mo>
-              </mover>
-            </mrow>
-          </mtd>
-          <mtd>
-            <mo>=</mo>
-          </mtd>
-          <mtd>
-            <mn>0</mn>
-          </mtd>
-        </mtr>
-      </mtable>
-    </mrow>
-  </mstyle>
-</math>
-</p>
-
-<h2>Einstein's field equations</h2>
-
-<p style="text-align:center">
-<math xmlns='&mathml;'>
-  <mstyle displaystyle='true'>
-    <mrow>
-      <msub>
-        <mi mathvariant="normal">R</mi>
-        <mstyle scriptlevel='1'>
-          <mrow>
-            <mi>&mu;</mi>
-            <mi>&nu;</mi>
-          </mrow>
-        </mstyle>
-      </msub>
-      <mo>-</mo>
-      <mfrac>
-        <mn>1</mn>
-        <mn>2</mn>
-      </mfrac>
-      <msub> 
-        <mi>g</mi>
-        <mstyle scriptlevel='1'>
-          <mrow>
-            <mi>&mu;</mi>
-            <mi>&nu;</mi>
-          </mrow>
-        </mstyle>
-      </msub>
-      <mi mathvariant='normal'>R</mi>
-      <mo>=</mo>
-      <mfrac>
-        <mrow>
-          <mn>8</mn>
-          <mi>&pi;</mi>
-          <mi mathvariant='normal'>G</mi>
-        </mrow>
-        <msup>
-          <mi>c</mi>
-          <mn>4</mn>
-        </msup>
-      </mfrac>
-      <msub>
-        <mi mathvariant='normal'>T</mi>
-        <mstyle scriptlevel='1'>
-          <mrow>
-            <mi>&mu;</mi>
-            <mi>&nu;</mi>
-          </mrow>
-        </mstyle>
-      </msub>
-    </mrow>
-  </mstyle>
-</math>
-</p>
-
-</body>
-</html>
